modules/consentManagement.js

```diff
@@ -336,18 +503,26 @@ function exitModule(errMsg, hookConfig, extraArgs) {
 export function resetConsentData() {
   consentData = undefined;
   gdprDataHandler.setConsentData(null);
+  ccpaDataHandler.setConsentData(null);
 }
 
 /**
  * A configuration function that initializes some module variables, as well as add a hook into the requestBids function
  * @param {object} config required; consentManagement module config settings; cmp (string), timeout (int), allowAuctionWithoutConsent (boolean)
  */
-export function setConsentConfig(config) {
-  if (utils.isStr(config.cmpApi)) {
-    userCMP = config.cmpApi;
-  } else {
-    userCMP = DEFAULT_CMP;
-    utils.logInfo(`consentManagement config did not specify cmp.  Using system default setting (${DEFAULT_CMP}).`);
+export function setConsentConfig(config, consentModule) {
+  if (consentModule) {
+    if (['gdpr', 'iab'].includes(consentModule)) {
+      userCMP = 'iab';
+    }
+
+    if (consentModule === 'static') {
+      userCMP = 'static';
+    }
+
+    if (!['gdpr', 'iab', 'static', 'usp'].includes(consentModule) && consentModule) {
+      userCMP = consentModule;
+    }
```

Comment:
and if we do want to do this, `if (!['gdpr', 'iab', 'static', 'usp']...` overwrites the first 2 `if` conditions.

Reply:
The last check is for a silly test that allows non-sense to be sent in. Technically I block that now.

If I leave the existing logic it will always instantiate GDPR

Reply:
I missed the `!` in the last `if`